bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityUIWidgetAddonHandler.java

```diff
@@ -86,7 +86,7 @@ public void install(Addon addon) throws MarketplaceHandlerException {
             } else if (yamlContent != null) {
                 addWidgetAsYAML(addon.getUid(), yamlContent);
             } else {
-                throw new IllegalArgumentException("Couldn't find the widget in the add-on entry");
+                throw new IllegalArgumentException("Couldn't find the widget in the add-on entry, or the starting code fence is not marked as ```yaml");
```

Comment:
```suggestion
                throw new IllegalArgumentException("Couldn't find the widget in the add-on entry. The starting code fence may not be marked as ```yaml");
```

It's not one or the other. It's a cause for not being able to find the widget.

Reply:
So what is the plan forwards? Reword it or ignore that there are. At least 4 widgets that can not be installed because of the missing yaml tag on the code fence? Wasted a few hours tracking the reason down because the message gave no clues as to the real cause.

Reply:
I think it's clear my suggestion is to reword it? It's a good a addition to have a common cause in the exception message. 👍